### ISSUE TITLE
docs CI does not run on renovate PRs

### ISSUE THREAD

**Issue by anyasabo** [elastic/cloud-on-k8s#3437]
To run Jenkins CI by default on renovate PRs, we had to whitelist the renovate[bot] user. It does not appear to be running docs builds though (for example https://github.com/elastic/cloud-on-k8s/pull/3417)

**Comment by alaudazzi** [comment]
@gtback would you mind looking into this?

**Comment by gtback** [comment]
Yep, this is pretty easy, though it would apply to all repos with Docs PR checks.
